apps/examples/App.tsx

```diff
@@ -133,27 +150,27 @@ const MainScreen = ({ navigation }: MainScreenProps): React.JSX.Element => (
     <Text style={styles.label} testID="root-screen-examples-header">
       Examples
     </Text>
-    {Object.keys(SCREENS)
-      .filter(name => SCREENS[name].type === 'example')
-      .map(name => (
-        <ListItem
-          key={name}
-          testID={`root-screen-example-${name}`}
-          title={SCREENS[name].title}
-          onPress={() => navigation.navigate(name)}
-        />
-      ))}
-    <Text style={styles.label}>Playgrounds</Text>
-    {Object.keys(SCREENS)
-      .filter(name => SCREENS[name].type === 'playground')
-      .map(name => (
-        <ListItem
-          key={name}
-          testID={`root-screen-playground-${name}`}
-          title={SCREENS[name].title}
-          onPress={() => navigation.navigate(name)}
-        />
-      ))}
+    {examples.map(name => (
+      <ListItem
+        key={name}
+        testID={`root-screen-example-${name}`}
+        title={SCREENS[name].title}
+        onPress={() => navigation.navigate(name)}
+      />
+    ))}
+    {!!playgrounds.length && (
```

Comment:
Should do the same with less nesting. 
```
{!!playgrounds.length && <Text style={styles.label}>Playgrounds</Text>}
         {playgrounds.map(name => (
           <ListItem
             key={name}
             testID={`root-screen-playground-${name}`}
             title={SCREENS[name].title}
             onPress={() => navigation.navigate(name)}
           />
         ))}
     )}
```

Reply:
https://github.com/software-mansion/react-native-screens/pull/2173/commits/5654ac07ae8004ca2c8d3a88531a36b9e29bfaa1